portfolio/src/main/webapp/index.html

```diff
@@ -168,11 +111,34 @@ <h3>Shpe</h3>
         </button>
     </div>
 
-    <footer> 
-        <h5>
-            Copyright of Sofi
-        </h5>
-    </footer>
+    <div class="comment-container">
+      <h1 id="Comment">Comments</h1>
+      <form method="POST" action="/data">
+        <label for="numbers">Choose the number of Comments displayed</label>
+        <!-- onchange will change whenever there is a change in the value -->
+            <select name="num" id="num" onchange="loadUpdatedComments()">
+                <option value="1">1</option>
+                <option value="3">3</option>
+                <option value="5">5</option>
+                <option value="8">8</option>
+                <option value="10">10</option>
+                <option value="12">12</option>
+                <option value="14">14</option>
+            </select>
+      </form>
 
+      <form method="POST" action="/data">
+      <h3>Name:</h3>
+      <input type="text" name="name" placeholder="type your name"/>
```

Comment:
For "input" and "br", we should get rid of the trailing "/" at the end of each element. That's an old practice that we don't do anymore.